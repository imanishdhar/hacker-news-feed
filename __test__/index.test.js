import React from 'react';
import { shallow } from 'enzyme';

import HomePage from '../pages';

function setup() {
  const wrapper = shallow(<HomePage></HomePage>);
  return { wrapper };
}

describe('HomePage Test Suite', () => {
  it('HomePage should render', () => {
    const wrapper = setup();
    expect(wrapper).toMatchSnapshot();
  });
});
