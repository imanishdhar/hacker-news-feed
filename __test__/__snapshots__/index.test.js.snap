// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage Test Suite HomePage should render 1`] = `
Object {
  "wrapper": ShallowWrapper {},
}
`;
